opentelemetry/src/metrics/instruments/mod.rs

```diff
@@ -95,34 +95,24 @@ impl<'a, T> HistogramBuilder<'a, T> {
 }
 
 impl<'a> HistogramBuilder<'a, Histogram<f64>> {
-    /// Validate the instrument configuration and creates a new instrument.
-    pub fn try_init(self) -> Result<Histogram<f64>> {
-        self.instrument_provider.f64_histogram(self)
-    }
-
     /// Creates a new instrument.
     ///
     /// Validates the instrument configuration and creates a new instrument. In
-    /// case of invalid configuration, an instrument that is no-op is returned
+    /// case of invalid configuration, a no-op instrument is returned
     /// and an error is logged using internal logging.
     pub fn init(self) -> Histogram<f64> {
```

Comment:
I believe this `init` method would be renamed to `build` as separate PR, as discussed in community meeting today?

Reply:
Yes. I'll do that in a follow-up PR.